content/en/news/releases/1.17.x/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: 1.17.x Releases
+description: Announcements for the 1.17 release and its associated patch releases.
+weight: 13
```

Comment:
Weight here goes to 12 (1 less than prior release).

Reply:
Done.

Reply:
As an FYI, this page has the release mixed up before the change. It's correct now (1.17 is first not second): https://deploy-preview-12633--preliminary-istio.netlify.app/latest/news/releases/